modules/coach/src/main/CoachPager.scala

```diff
@@ -48,6 +48,8 @@ final class CoachPager(
                     case Login         => Descending("user.seenAt")
                   }
                 ),
+                Skip(offset),
```

Comment:
if you skip first, and only then match the user country, then I don't think it's going to work out. The skip should only be after the filter. It's very inneficient, but it's correct